jsk_spot_watch_dog/scripts/watch_dog.py

```diff
@@ -0,0 +1,212 @@
+#!/usr/bin/env python
```

Comment:
Can we write in euslisp? This demo will require several type of robot motion to express its emotion in future. That’s would be good example of spoteus and it is always better to think how to use other’s work. This node requires P1, infra360, coral , that’s nice. If you use roseus, that’s much better.

Reply:
Yes, it would be a good practice. I will write in euslisp